### DISCUSSION TITLE
Add speak text / dictate interaction

### DISCUSSION THREAD

**Discussion by jeby** [Yue-Yang/ChatGPT-Siri#38] [Ideas]
Hi, I've seen some other shortcuts trying to integrate chatGPT and Siri (i.e. Siri Pro) having a voice interaction, more similar to what regular Siri is doing. Maybe you can try to add this functionality and streamline the interaction.

**Comment by Yue-Yang** [comment]
You can use that one instead. My shortcuts have different logic and support all apple devices.
